src/librustc_resolve/imports.rs

```diff
@@ -262,8 +262,8 @@ impl<'a> Resolver<'a> {
         }
 
         let check_usable = |this: &mut Self, binding: &'a NameBinding<'a>| {
-            if let Some(blacklisted_binding) = this.blacklisted_binding {
-                if ptr::eq(binding, blacklisted_binding) {
+            if let Some(blocklisted_binding) = this.blocklisted_binding {
```

Comment:
I wouldn't personally want to replace an idiomatic and understandable name with a niche newspeak.

If you can come up something that is an actual word that means "a binding that should not be considered as a candidate because otherwise it would cause a cyclic dependency", then it can be used instead if the majority of compiler team agrees that some color names shouldn't be used when naming variables.

(I won't argue about the underlying premises that don't make sense to me, because it's not a place to do that and bringing *more* politics is the last thing that I'd want to do.)